kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/utils/SqlUtils.java

```diff
@@ -90,6 +102,36 @@ private static String getCreateTableQuery(String datasetName, LinkedHashMap<Stri
         return createTableQuery.toString();
     }
 
+    private static LinkedHashMap<String, VariableType> getVariablesToAdd(
+            Statement statement,
+            String datasetName,
+            LinkedHashMap<String, VariableType> sqlSchema
+    ) throws SQLException {
+        LinkedHashMap<String,VariableType> variablesToAdd = new LinkedHashMap<>();
+        List<String> columnsAlreadyInDatabase = getColumnNames(statement, datasetName);
+        //Filter out variable names already in database table
+        for(String variableNameToAdd :
```

Comment:
sqlSchema.keySet().stream()
        .filter(variableName -> !columnsAlreadyInDatabaseSet.contains(variableName))
        .forEach(variableNameToAdd -> variablesToAdd.put(variableNameToAdd, sqlSchema.get(variableNameToAdd))); 
    
to avoid "for" and keep the stream